main.py

```diff
@@ -50,30 +51,30 @@ def run(
     print(f'[bold cyan]🚀 Starting benchmark run: {run_number}/{runs}...[/bold cyan]')
 
     # Go to URL
-    page.goto(url)
+    await page.goto(url)
 
     # Focus on and click the map element
-    page.focus('.mapboxgl-canvas')
-    page.click('.mapboxgl-canvas')
+    await page.focus('.mapboxgl-canvas')
+    await page.click('.mapboxgl-canvas')
     # use performance.mark API to mark the start of the benchmarks.
-    page.evaluate(
+    await page.evaluate(
```

Comment:
This is a case where we'd want to fire this off synchronously, right?
```suggestion
    page.evaluate(
```

Reply:
good catch. for these two operations to be executed at the same time, the following should work

```python
    await asyncio.gather(
        page.evaluate(
            """
            () => (window.performance.mark("benchmark:start"))
                      """
        ),
        page.click('//div[text()="Display"]/following-sibling::button')
    )

```